editor/osm_auth.cpp

```diff
@@ -0,0 +1,418 @@
+#include "editor/osm_auth.hpp"
+
+#include "coding/url_encode.hpp"
+
+#include "base/assert.hpp"
+#include "base/logging.hpp"
+
+#include "std/iostream.hpp"
+#include "std/map.hpp"
+
+#include "private.h"
+
+#include "3party/liboauthcpp/include/liboauthcpp/liboauthcpp.h"
+#include "3party/Alohalytics/src/http_client.h"
+
+using alohalytics::HTTPClientPlatformWrapper;
+
+namespace osm
+{
+constexpr char const * kApiVersion = "/api/0.6";
```

Comment:
No, it's not. As we found out constexpr have special meaning for const char *.
And works something like a macro.